apps/cyberstorm-storybook/package.json

```diff
@@ -40,6 +40,7 @@
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-typescript": "^7.16.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```

Comment:
Not sure if this change would result in changes in `yarn.lock`, but if it does, it would be good practice to include the changes to lock file in the same commit. Not a huge deal since it's fixed in the next commit (and these commits could be squashed).

Reply:
Yeah, I'm not sure either. I'll remove this and check all the package versions relevant to my components in another separate PR.

Reply:
Seems it's still there? And yarn.lock wasn't updated in the latest commit?